ClosedXML/ClosedXML.csproj

```diff
@@ -15,12 +15,13 @@
     <Copyright>MIT</Copyright>
     <PackageProjectUrl>https://github.com/ClosedXML/ClosedXML</PackageProjectUrl>
     <PublishRepositoryUrl>true</PublishRepositoryUrl>
-    <PackageIconUrl>https://raw.githubusercontent.com/ClosedXML/ClosedXML/develop/resources/logo/nuget-logo.png</PackageIconUrl>
+    <PackageIconUrl></PackageIconUrl>
```

Comment:
Please rather do this in a separate PR.

Reply:
Sure.
I created new pr with this fix - https://github.com/ClosedXML/ClosedXML/pull/1516.